android-agent/src/main/java/io/opentelemetry/android/SessionId.java

```diff
@@ -33,7 +33,7 @@ class SessionId {
         this.clock = clock;
         this.timeoutHandler = timeoutHandler;
         value.set(createNewId());
-        createTimeNanos = clock.now();
+        createTimeNanos = clock.nanoTime();
```

Comment:
Per doc
>>
Returns the current epoch timestamp in nanos from this clock. This timestamp should only be used to compute a current time. To compute a duration, timestamps should always be obtained using nanoTime(). For example, this usage is correct.
 long startNanos = clock.nanoTime();
 // Spend time...
 long durationNanos = clock.nanoTime() - startNanos;
 
This usage is NOT correct.
 long startNanos = clock.now();
 // Spend time...
 long durationNanos = clock.now() - startNanos;